src/NServiceBus.Core/Recoverability/IRecoverabilityActionContext.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace NServiceBus.Pipeline
```

Comment:
Should this be `NServiceBus.Recoverability` ?

Reply:
All context interfaces are in `NServiceBus.Pipeline`